Makefile

```diff
@@ -29,8 +29,10 @@ clean-pyc:
 	find . -name '*~' -exec rm -f {} +
 
 clean-docs:
-	rm -f docs/kolibri*rst
-	rm -f docs/modules.rst
+	rm -f docs/py_modules/kolibri*rst
+	rm -f docs/py_modules/modules.rst
+	rm -f docs/kolibri*rst # old location
```

Comment:
I left them in case files still exist in the old location for some devs. My hope was that this would clean them up, and sometime in the future we can remove the older references. Thoughts?